Minor fixes on image link and syntax enhancements (#47)
jtpio/jupyterlab-extension-examples: widget-tracker/widgets/README.md

# Widgets - Adding new Elements to the Main Window

We are going to do some real stuff and add a new tab to JupyterLab.

> Visible elements such as a tab are represented by widgets in the phosphor library

Reviewer: ```suggestion
Visible elements such as tabs and notebooks are represented by widgets in the Phosphor library
```
